### ISSUE TITLE
[Bug] - Stage areas code don't detection change record.

### ISSUE THREAD

**Issue by yaolifeng0629** [yaolifeng0629/easy-commit-util#7]
![image](https://github.com/yaolifeng0629/easy-commit-util/assets/88535417/d1275d40-d26d-4dc5-8dea-8cc0de6ee89b)

**Comment by yaolifeng0629** [comment]
```js
function isModify() {
    // Check temporary and staging area modifications
    const resultUnstaged = execSync('git diff --name-only').toString();
    const resultStaged = execSync('git diff --cached --name-only').toString();

    // List of modified files merging temporary and staging areas
    const filesModified = (resultUnstaged + '\n' + resultStaged)
        .split('\n')
        .map(line => line.trim())
        .filter(file => file);

    return filesModified.length > 0;
}
```

**Comment by yaolifeng0629** [comment]
Fixed，to See #8
